components/salesforce/sources/object-deleted-instant/object-deleted-instant.js

```diff
@@ -0,0 +1,43 @@
+const startCase = require("lodash/startCase");
+
+const common = require("../../common-instant");
+
+module.exports = {
+  ...common,
+  name: "Object Deleted (Instant)",
+  key: "salesforce-object-deleted-instant",
+  description: "Emit an event when an object is deleted",
+  version: "0.0.1",
+  methods: {
+    ...common.methods,
+    isValidSObject(sobject) {
```

Comment:
This logic isn't relevant for new objects, correct?

Reply:
You are correct, this is something I forgot to remove.